### ISSUE TITLE
Unexpected behaviour when parsing lists in INFO field

### ISSUE THREAD

**Issue by holtgrewe** [samtools/htsjdk#261]
I have a VCF file with an `INFO` field that has `A` many values, but these values can be empty, as demonstrated below. Reading this with HTSJDK leads to the `FI` and `FS` fields having one atomic value instead of a list of two values.

Is this intented behaviour?

```
1       69360   .       C       A,G     .       .       RV=C>*,C>W;GN=OR4F5;UG=OR4F5_HUMAN;VI=stop_gain,;UV=C90*,C90W;FI=,high;FS=,4.765
```

**Comment by yfarjoun** [comment]
going over old issues...

The missing string should be the empty string, but it isn't clear the the double array is legal since the spec doesn't seem to allow for missing individual fields in an array. so that would have to be an error.

I think that if strict pasrsing is requested, this record should be deemed not conformant due to the float array, and the string array should have returned one empty string.

**Comment by holtgrewe** [comment]
Thanks for considering this issue.

I don't see in the specs where empty strings are disallowed.

**Comment by yfarjoun** [comment]
I agree.. an empty string should have been created, but not a missing float

**Comment by tfenne** [comment]
Fixed in 1.133 by #248: INFO values are split with a splitter that keeps empty tokens, so `FI=,high` parses as the two-element list `["", "high"]`.
